pkg/oc/admin/diagnostics/diagnostics/cluster/scc.go

```diff
@@ -0,0 +1,95 @@
+package cluster
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	kerrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/apis/authorization"
+	authorizationtypedclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/authorization/internalversion"
+
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	"github.com/openshift/origin/pkg/oc/admin/diagnostics/diagnostics/types"
+	"github.com/openshift/origin/pkg/oc/admin/diagnostics/diagnostics/util"
+	policycmd "github.com/openshift/origin/pkg/oc/admin/policy"
+	securityapi "github.com/openshift/origin/pkg/security/apis/security"
+	securitytypedclient "github.com/openshift/origin/pkg/security/generated/internalclientset/typed/security/internalversion"
+)
+
+// SCC is a Diagnostic to check that the default securitycontextconstraints are present and contain the expected permissions.
+type SCC struct {
+	SCCClient securitytypedclient.SecurityContextConstraintsInterface
+	SARClient authorizationtypedclient.SelfSubjectAccessReviewsGetter
+}
+
+const SccName = "SCC"
```

Comment:
Shouldn't we use full resource name and additionally allow using shorter aliases? That's the approach we're taking across the entire CLI, I don't think this should be any different.

Reply:
I see. I basically agree that full resource name with shorter alias.

However, subcommands of diagnostics set their alias in one same code here:

https://github.com/openshift/origin/blob/e9277a4c9f09115a13a719c78afef0ef90736ed8/pkg/oc/admin/diagnostics/diagnostics.go#L179

So, if I try to let it use `securitycontextcontraints` with `scc` alias, I need to change above code. An all of subcommands of diagnostics need to be updated a little bit.

Reply:
I will defer these CLI specifics to @soltysh as I have no strong opinions.

Reply:
@nak3 then just do full name (like with `oc adm diagnostics clusterrolebindings`) and don't mention short names at all.

Reply:
@soltysh Sure, updated.